ViewTemplates/Sites/default.blade.php

```diff
@@ -160,6 +160,39 @@ class="btn btn-primary">
         </tfoot>
     </table>
 
+    <!-- Modal -->
+    <div class="modal fade" id="batchModal">
+        <div class="modal-dialog modal-lg">
+            <div class="modal-content">
+                <div class="modal-header">
+                    <h5 class="modal-title" id="batchModalLabel">@lang('PANOPTICON_SITES_BATCH_TITLE')</h5>
+                    <button type="button" class="btn-close" data-bs-dismiss="modal" aria-label="Close"></button>
```

Comment:
aria-label should be translatable

Reply:
Made it translatable. Thank you for the heads up!